transport/transport.go

```diff
@@ -0,0 +1,17 @@
+/*
+Package transport holds the Transporter interface which is used by the Zipkin
+Tracer to send finished spans.
+
+Subpackages of package transport contain officially supported standard
+transport implementations.
+*/
+package transport
+
+import "github.com/openzipkin/zipkin-go/model"
+
+// Transporter interface can be used to provide the Zipkin Tracer with custom
+// implementations to publish Zipkin Span data.
+type Transporter interface {
```

Comment:
in our arch etc we use the term reporter or sender, sadly javascript calls this logger, and census exporter. transporter is another name to add to the list :) I do see why you used it.

Reply:
There is much to be said for consistency and established terminology. I'm not happy with adding a new term but zipkin-go-opentracing unfortunately used Collector (which is the other side) so that's why I changed it to Transporter. If you agree I'd like to change to `Reporter`